src/Simulation/Native/src/CMakeLists.txt

```diff
@@ -34,9 +34,40 @@ else(BUILD_SHARED_LIBS)
   set_source_files_properties(simulator/simulatoravx512.cpp PROPERTIES COMPILE_FLAGS ${AVX512FLAGS})
 endif(BUILD_SHARED_LIBS)
 
-target_link_libraries(Microsoft.Quantum.Simulator.Runtime ${SPECTRE_LIBS})
+target_include_directories(Microsoft.Quantum.Simulator.Runtime PRIVATE "${PROJECT_BINARY_DIR}/../../../Qir/drops/include")
+
+if (WIN32)
+  target_link_libraries(Microsoft.Quantum.Simulator.Runtime
+    ${SPECTRE_LIBS}
+    "-L${PROJECT_BINARY_DIR}/../../../Qir/drops/bin/$ENV{BUILD_PLATFORM}/native"
+    qir_stdlib
+    Ws2_32
+    Bcrypt
+    Userenv
+  )
+  # On Windows, use a .def file to force export stdlib functions from the static library.
+  target_sources(Microsoft.Quantum.Simulator.Runtime PUBLIC "${PROJECT_BINARY_DIR}/../../../Qir/drops/include/qir_stdlib.def")
+elseif (APPLE)
+  # On MacOS, use -force_load and explicit path to pull in entire contents of stdlib static archive.
```

Comment:
Inlcude links for Windows, Mac, and Linux static linking strategies (flags definitions where available)